### PR TITLE
Real-time update chart for activity-monitor project

### PR DESCRIPTION
**Pull request by lbarczynski**

### CODE REVIEW

**Comment by zeke** [comment]
Thanks! Can you summarize the changes you've made here?

**Comment by lbarczynski** [comment]
I added simple "udateDatasets" listener, runned with 1000ms interval. Datasets are updated according to documentation:

http://www.chartjs.org/docs/latest/developers/updates.html

Problem is that diffrence between two measurements are too small to show real CPU consumtion (especially for long-time running machine), so I save last CPU times and return only difference for new and old measurements. That pseudo-code should explain all changes:

`
var lastCpuTimes;
while(true){
    var currentCpuTimes = getCpuTimes();
    updateDatasets(currentCpuTimes - lastCpuTimes);
   lastCpuTimes = currentCpuTimes;
   wait(1000);
}
`

**Comment by zeke** [comment]
Gave it a try locally. Looks good! 👍
